docs(theme): updated related docs with adding font family (#1999)
pankod/refine: documentation/docs/ui-frameworks/mui/tutorial.md

`<GlobalStyles>` is a component that is used to apply the global styles to the application. You may refer to the [**GlobalStyles** documentation](https://mui.com/material-ui/api/global-styles/) for more information.
:::

:::info

> Font family comes in the theme as specified in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property. Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) font as default. You have to embed to `index.html` if you want to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) as font family.

Reviewer: ```suggestion
Font family comes in the theme as specified in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property. Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) font as default. You have to embed to `index.html` if you want to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) as the font family.
```

> Font family comes in the theme as specified in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property. Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) font as default. You have to embed to `index.html` if you want to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) as font family.

Reviewer 2: ```suggestion
Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat) font by default as it is defined in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property of the theme.  But to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat), you need to embed it to your `index.html` file.
```

> For more information about to add font family in your Refine application, you can look at [`Material UI Theme Customization`](/ui-frameworks/mui/customization/theme.md#overriding-variables).

Reviewer: ```suggestion
For more information about adding font family in your Refine application, you can look at [`Material UI Theme Customization`](/ui-frameworks/mui/customization/theme.md#overriding-variables).
```

```html title="index.html"
<!DOCTYPE html>
<html lang="en">
    <head>
        // highlight-start
        <link
            href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap"
            rel="stylesheet"
        />
        // highlight-end
        <title>refine adding font family example</title>
    </head>

    <body>
        ...
    </body>
</html>
```

:::

:::tip

> **refine** comes natively with Light/Dark theme support. Check out the [theme](#) documentation for more information.

Reviewer 2: And we have a missing link here 🤔
